protected/humhub/modules/content/services/ContentSearchService.php

```diff
@@ -38,9 +39,9 @@ public function update($asActiveJob = true): void
         }
     }
 
-    public function delete($asActiveJob = true): void
+    public function delete(bool $asActiveJob = true, bool $forceDelete = false): void
```

Comment:
@yurabakhtin Not sure about the `forceDelete` option. I would prefer to remove `isIndexable()` here for all cases.

What do you think about?